config/environments/demo.rb

```diff
@@ -38,5 +38,5 @@
   config.ctc_full_launch = Time.find_zone("America/New_York").parse("2022-04-01 09:00:00")
 
   # StateFile
-  config.state_file_start_of_open_intake = Time.find_zone('America/New_York').parse('2024-01-01 7:59:59')
```

Comment:
you'll have to change this in `config/environments/staging.rb` instead, this will change the config time for the demo environment @rickreyhsig

Reply:
Yep, I realized I was conflating the two envs when I opened this PR @embarnard